tests/Hook/DoubleWriteHookTest.php

```
@@ -160,4 +163,167 @@ public function testAfterWriteDoesNotThrowExceptionWhenFailOnSecondaryErrorIsFal
+        // storageは使われない（エラーでクリーンアップされるため）
+        $mockStorage->expects(self::never())
+            ->method('set');
+
+        $logger = $this->createMock(LoggerInterface::class);
+        $logger->expects(self::once())
+            ->method('error')
+            ->with('Primary write error, secondary write skipped', self::anything());
+
+        $hook = new DoubleWriteHook($this->secondaryConnection, 1440, false, $logger);
+        $hook->beforeWrite('test_session', ['key' => 'value'], $mockStorage);
+        $hook->onWriteError('test_session', new \Exception('Test error'));
```

PHPMD (2.15.0)

Missing class import via use statement (line '283', column '49'). (undefined)

(MissingImport)